stream_alert_cli_module/runner.py

```diff
@@ -604,6 +596,8 @@ def configure_output(options):
     props = output.get_user_defined_properties()
 
     for name, prop in props.iteritems():
+        # TODO: Don't use protected method here
+        # pylint: disable=protected-access
```

Comment:
Actually, I'll just leave it as is. It's very convenient, and people do seem to use it. Not sure why Python wouldn't expose it as a normal method